lib/stack_master/aws_driver/cloud_formation.rb

```diff
@@ -51,6 +53,23 @@ def validate_template(options)
       def cf
         @cf ||= Aws::CloudFormation::Client.new(region: @region)
       end
+
+      def retry_with_backoff
+        delay       = 1
+        max_delay   = 30
+        begin
+          yield
+        rescue Aws::CloudFormation::Errors::Throttling => e
+          if e.message =~ /Rate exceeded/
+            sleep delay
```

Comment:
Ehhh, I see both sides of it.  Here are my thoughts:

Pro:
* I like transparency
* Blames AWS for slowness, not stackmaster (yay blame!)

Con:
* Not actionable info to the user in any way (currently).
* Screen clutter

In my particular use case we have quite a lot of stacks, so this happens frequently, and beyond the first time it happens the novelty is lost.  Hence I opted for silence.  But I don't feel strongly about it, so as always, feedback is welcome.